src/capitalsFirst.js

```diff
@@ -18,7 +18,14 @@
  * @returns {string} - string with uppercase words in front
  */
 function capitalsFirst(str) {
-  // write code here
+  const onlyLettersFirst = str.split(' ').filter(item =>
```

Comment:
why do you filter it?

Reply:
> why do you filter it?

because according to the task result string must be without words which start with not a letter - so i filtered these words